demo/lib/distro.bash

```diff
@@ -744,6 +744,7 @@ touch /etc/sysctl.d/k8s.conf
 echo "net.bridge.bridge-nf-call-ip6tables = 1" >> /etc/sysctl.d/k8s.conf
 echo "net.bridge.bridge-nf-call-iptables = 1" >> /etc/sysctl.d/k8s.conf
 echo "net.ipv4.ip_forward = 1" >> /etc/sysctl.d/k8s.conf
+echo "net.ipv4.conf.*.rp_filter = 0" >> /etc/sysctl.d/k8s.conf
```

Comment:
Even narrower, limited to lxc interfaces should be enough:
```
net.ipv4.conf.lxc*.rp_filter = 0
```

Reply:
@askervin: what do you think about this? In our use case only the lxc interfaces need this. I'm fine with the little bit bigger hammer, too

Reply:
@marquiz , in this case bigger hammer feels better to me. I'm more concerned on hard-to-debug `rp_filter`ing issues on various CNI and container runtime implementations than malicious IP spoofing in our e2e VMs.

(Even if this is now documented and spelled out loud, someone will anyway copy-paste the setup routine to a life-critical cluster setup that will then be DDOSsed, but that's how things go because of  Murphy's law...)